manual/create-repo.md

```diff
@@ -1,44 +1,134 @@
+            * **Require branches to be up-to-date before merging** *(sub option)*
+                * **Note**: the GitHub Actions job will need to be configured first before this status check will
+                  appear.
+    * **Require signed commits**
     * **Include administrators**
-    
-    For any repositories that use a multi-branch Jenkins pipeline the following additional branch protection rules should be set for the *master* rule:
-    * **Require status checks to pass before merging**      
-      * **Require branches to be up to date before merging** *(sub option)*
-      
-      Choose the following status checks:
-      * **TDR Jenkins build status** *(make this a required check)*
-      
-      **Note**: the Jenkins job will need to be configured first before this status check will appear. For example see: https://github.com/nationalarchives/tdr-transfer-frontend/blob/master/Jenkinsfile-testing, where the reporting functions are called (`tdr.reportStartOfBuildToGitHub`, `tdr.reportFailedBuildToGitHub`, `tdr.reportSuccessfulBuildToGitHub`)  
-      
-* Add an open source licence
-  * For code repos, add an MIT licence ([example][mvc-licence])
-  * For documentation, add an MIT licence for the code and an [Open Government
-    Licence][ogl] for the content. See this repo's LICENCE file the licence note
-    in the README for an example.
-* Add a link from the new project's README to this documentation, to add context
-  for anyone who finds the project
-* Add the new project to the repo list in the README of this project
-* If the project contains a [dependabot supported package manager][supported-package-managers] then add a [dependabot config file][dependabot-config].
-* If the project is a Scala project, add the repository to the [list of repos maintained by Scala Steward].
+* Click the "Save changes" button
+
+### 6. Add workflow files
+
+* In the repo, if there isn't a `.github` directory create one
+* Within that `.github` directory, create a `workflows` directory
+* copy the `build.yml`, `deploy.yml` and `test.yml` from a similar repo ([example][example-repo-workflow-files])
+* For the build.yml
+    * replace all instances of the older repo's name with the name of your repo
+    * under `pre-deploy: > uses` change the [workflow][workflow-files] to the "build" workflow relevant for your repo
+    * change the build command to run whatever commands the project needs to build
+* For the deploy.yml
+    * under `deploy: > uses` change the [workflow][workflow-files] to the "deploy" workflow relevant for your repo
+* For the test.yml
+    * under `jobs: > test: > with: > test-command: >` change the build test-commands to run whatever commands the
+      project needs to test its code
+* Once this is done, commit the files and push them to the repo; you will notice that the test job will start running
+    * **Note**: In order to push to the repo, you'll need to create a new branch
+
```

Comment:
Should include a section on adding the environment secrets if they are needed by the actions. These are set in the terraform.

Some notes from @MancunianSam about this:

> The environment secrets are defined in https://github.com/nationalarchives/tdr-terraform-environments/blob/master/root_github.tf#L407 You can copy the existing lambdas in there.
> The repository secrets are defined in https://github.com/nationalarchives/tdr-terraform-backend/blob/master/root_github.tf#L559
> You can set them manually if you want to test them, terraform will overwrite them when it gets run.